Remove AppleWin code from DLLLauncherMain.cpp

fujii opened the pull request:
#### bd7aade58f9bbfc59da1f932823380c8cf3d18dc
<pre>
Remove AppleWin code from DLLLauncherMain.cpp
<a href="https://bugs.webkit.org/show_bug.cgi?id=252853">https://bugs.webkit.org/show_bug.cgi?id=252853</a>

Reviewed by Don Olmstead.

* Source/JavaScriptCore/shell/DLLLauncherMain.cpp:
(modifyPath):
(getStringValue): Deleted.
(applePathFromRegistry): Deleted.
(appleApplicationSupportDirectory): Deleted.
(iTunesDirectory): Deleted.
(prependPath): Deleted.
* Source/JavaScriptCore/shell/PlatformWin.cmake:
* Source/WebDriver/PlatformWin.cmake:
* Tools/ImageDiff/PlatformWin.cmake:
* Tools/MiniBrowser/win/CMakeLists.txt:
* Tools/TestWebKitAPI/PlatformWin.cmake:
* Tools/WebKitTestRunner/PlatformWin.cmake:
* Tools/win/DLLLauncher/DLLLauncherMain.cpp:
(modifyPath):
(getStringValue): Deleted.
(applePathFromRegistry): Deleted.
(appleApplicationSupportDirectory): Deleted.
(iTunesDirectory): Deleted.
(prependPath): Deleted.

Canonical link: <a href="https://commits.webkit.org/260768@main">https://commits.webkit.org/260768@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/da9739a1b87fe14b988ed9a43ec573110dde6a39

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/109334 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/18413 "Built successfully") | [  ~~🛠 mac~~](https://ews-build.webkit.org/#/builders/43/builds/42142 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wpe~~](https://ews-build.webkit.org/#/builders/87/builds/857 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/12/builds/118540 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/113217 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/19852 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/9693 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/101571 "Built successfully") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/115090 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/78/builds/14879 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/43/builds/42142 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/35/builds/43064 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/9/builds/96851 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/43/builds/42142 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/34/builds/84814 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [  ~~🛠 🧪 jsc~~](https://ews-build.webkit.org/#/builders/1/builds/98300 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/11180 "Built successfully") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/43/builds/42142 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 jsc-armv7~~](https://ews-build.webkit.org/#/builders/38/builds/99195 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [  ~~🛠 🧪 jsc-arm64~~](https://ews-build.webkit.org/#/builders/83/builds/9249 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/82/builds/11916 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/84/builds/8036 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 jsc-armv7-tests~~](https://ews-build.webkit.org/#/builders/38/builds/99195 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/80/builds/17283 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/43/builds/42142 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 jsc-mips~~](https://ews-build.webkit.org/#/builders/37/builds/107160 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/7435 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/13537 "Built successfully") | | [  ~~🧪 jsc-mips-tests~~](https://ews-build.webkit.org/#/builders/37/builds/107160 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | | | | 
<!--EWS-Status-Bubble-End-->

donny-dont commented on Tools/WebKitTestRunner/PlatformWin.cmake:
```diff
@@ -10,7 +10,6 @@ list(APPEND WebKitTestRunner_SOURCES
 
 set(wrapper_DEFINITIONS
```
Nit feel free to ignore but maybe turn into a one liner for consistency.

fujii replied:
Fixed.

donny-dont replied:
Actually looking a bit more you could probably get rid of the `wrapper_DEFINITIONS`

fujii committed:
Remove AppleWin code from DLLLauncherMain.cpp
https://bugs.webkit.org/show_bug.cgi?id=252853

Reviewed by Don Olmstead.

* Source/JavaScriptCore/shell/DLLLauncherMain.cpp:
(modifyPath):
(getStringValue): Deleted.
(applePathFromRegistry): Deleted.
(appleApplicationSupportDirectory): Deleted.
(iTunesDirectory): Deleted.
(prependPath): Deleted.
* Source/JavaScriptCore/shell/PlatformWin.cmake:
* Source/WebDriver/PlatformWin.cmake:
* Tools/ImageDiff/PlatformWin.cmake:
* Tools/MiniBrowser/win/CMakeLists.txt:
* Tools/TestWebKitAPI/PlatformWin.cmake:
* Tools/WebKitTestRunner/PlatformWin.cmake:
* Tools/win/DLLLauncher/DLLLauncherMain.cpp:
(modifyPath):
(getStringValue): Deleted.
(applePathFromRegistry): Deleted.
(appleApplicationSupportDirectory): Deleted.
(iTunesDirectory): Deleted.
(prependPath): Deleted.

Canonical link: https://commits.webkit.org/260768@main
